docs/images/example-arch.plantuml

```diff
@@ -0,0 +1,42 @@
+@startuml
```

Comment:
You can use the following rendering online resource to edit and discover this file:

https://www.planttext.com/

For insights on syntax:  https://plantuml.com/component-diagram

Reply:
Nice way to create diagrams with human readable dsl.

Reply:
Glad that you liked it.

Most the IDEs have good plugins for live-rendering while editing the source code. Even though that can be done in the browser, I like to stay in the flow in the IDE. Here below screenshot of the plugin I use in Goland:

![image](https://user-images.githubusercontent.com/5781339/120934426-a197b900-c6fe-11eb-8cf3-f335eca17f41.png)

If you are interested here is the [link for the plugin home page](https://plugins.jetbrains.com/plugin/7017-plantuml-integration)